pom.xml

```diff
@@ -55,5 +55,10 @@
             <artifactId>aws-credentials</artifactId>
             <version>1.23</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>structs</artifactId>
```

Comment:
Previously picked up `@Symbol` from `jenkins-core`, which is dangerous for class loading reasons IIRC.